Code/src/main/java/nl/utwente/viskell/ghcj/ClashEvaluator.java

```diff
@@ -0,0 +1,24 @@
+package nl.utwente.viskell.ghcj;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+/**
+ * Evaluator for ghci that pushes code to the ghci environment and can evaluate expressions and return a result.
+ */
+public class ClashEvaluator extends PipeEvaluator {
+    public ClashEvaluator() throws HaskellException {
+        super();
+    }
+
+    @Override
+    protected List<String> getCommand() {
+        return ImmutableList.of("clash", "--interactive", "-ignore-dot-ghci");
+    }
+
+    @Override
+    protected List<String> getModules() {
+        return ImmutableList.of("+", "Data.List", "Data.Maybe", "Data.Either");
```

Comment:
"+" isn't exactly a module. :stuck_out_tongue:  I think it's better to put CLaSH.Prelude there*, so this is a list we can use to reload all the necessary modules (statelessly instead of relying on the ghci session) in the future when we get to adding and deleting module nodes.

*or enable a switch in CλaSH that implicitly loads its Prelude like GHCi does.
*or just move the "+" to the ":module " command.